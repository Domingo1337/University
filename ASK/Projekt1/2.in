mthi $t1		#różnoargumentowe instrukcje
mtlo $s2
mflo $s3
lui $t3, 0x1234
mult $3, $4
divu $5, $7
