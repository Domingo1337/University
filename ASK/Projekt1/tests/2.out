.text
00000000	01200011	mthi	$9
00000004	02400013	mtlo	$18
00000008	00009812	mflo	$19
0000000c	3c0b1234	lui	$11,0x1234
00000010	00640018	mult	$3,$4
00000014	00a7001b	divu	$5,$7
